packages/eas-cli/src/credentials/ios/actions/SetupBuildCredentials.ts

```diff
@@ -127,6 +127,12 @@ export class SetupBuildCredentialsFromCredentialsJson implements Action {
       throw error;
     }
 
+    if (isCredentialsMap(localCredentials)) {
```

Comment:
Update in another direction should also be blocked, technically current implementation works, but it's potentially breaking users configs if they run it